attack/scripts/bashrc

```diff
@@ -249,6 +249,14 @@ welcome() {
   info "  \$ next_hint"
   info "  \$ show_hints"
   info ' '
+  info 'To end a task, you can start a new task or:'
```

Comment:
welcome message should maybe mention that you can only finish a task once. 
also should you be able to do task 1 after task 2? In all cases task two will give the solution to task 1 and in most cases preclude it by preventing it